avm/res/digital-twins/digital-twins-instance/endpoint--service-bus/main.bicep

```diff
@@ -0,0 +1,90 @@
+metadata name = 'Digital Twins Instance ServiceBus Endpoint'
+metadata description = 'This module deploys a Digital Twins Instance ServiceBus Endpoint.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Optional. The name of the Digital Twin Endpoint.')
+param name string = 'ServiceBusEndpoint'
+
+@description('Conditional. The name of the parent Digital Twin Instance resource. Required if the template is used in a standalone deployment.')
+param digitalTwinInstanceName string
+
+@allowed([
+  'IdentityBased'
+  'KeyBased'
+])
+@description('Optional. Specifies the authentication type being used for connecting to the endpoint. If \'KeyBased\' is selected, a connection string must be specified (at least the primary connection string). If \'IdentityBased\' is selected, the endpointUri and entityPath properties must be specified.')
+param authenticationType string = 'IdentityBased'
+
+@description('Optional. Dead letter storage secret for key-based authentication. Will be obfuscated during read.')
+@secure()
+param deadLetterSecret string = ''
+
+@description('Optional. Dead letter storage URL for identity-based authentication.')
+param deadLetterUri string = ''
+
+@description('Optional. The URL of the ServiceBus namespace for identity-based authentication. It must include the protocol \'sb://\' (e.g. sb://xyz.servicebus.windows.net).')
+param endpointUri string = ''
+
+@description('Optional. The ServiceBus Topic name for identity-based authentication.')
+param entityPath string = ''
+
+@description('Conditional. PrimaryConnectionString of the endpoint for key-based authentication. Will be obfuscated during read. Required if the `authenticationType` is "KeyBased".')
+@secure()
+param primaryConnectionString string = ''
+
+@description('Optional. SecondaryConnectionString of the endpoint for key-based authentication. Will be obfuscated during read. Only used if the `authenticationType` is "KeyBased".')
+@secure()
+param secondaryConnectionString string = ''
+
+@description('Optional. The managed identity definition for this resource.')
+param managedIdentities managedIdentitiesType
+
+var identity = !empty(managedIdentities) ? {
```

Comment:
Same comment like in `endpoint--event-hub` regarding a potential warning.

Reply:
updated